modules/oe_content_sub_entity/modules/oe_content_sub_entity_document_reference/tests/src/Functional/DocumentReferenceBundlesTest.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_sub_entity_document_reference\Functional;
+
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\file\Entity\File;
+use Drupal\file\FileInterface;
+use Drupal\media\Entity\Media;
+use Drupal\media\MediaInterface;
+use Drupal\node\Entity\NodeType;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\field\Traits\EntityReferenceTestTrait;
+use Drupal\Tests\TestFileCreationTrait;
+
+/**
+ * Tests the document reference bundles defined in this module.
+ */
+class DocumentReferenceBundlesTest extends BrowserTestBase {
+
+  use EntityReferenceTestTrait;
+  use TestFileCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'node',
+    'oe_content_sub_entity_document_reference',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'classy';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Create a node type.
+    NodeType::create([
+      'type' => 'page',
+      'name' => 'Basic page',
+    ])->save();
+    $this->createEntityReferenceField(
+      'node',
+      'page',
+      'node_docs',
+      // Make the title fully distinguishable from other documents fields.
+      'Node docs',
```

Comment:
At the time I did this, the goal was to distinguish from labels within the document reference entity subforms, if we use label-based selectors. E.g. "Documents" may also refer to the list of documents within a document group.
Now we are using more targeted selectors, but I like to feel a bit safer.

> I would have called it anyway "Document groups".

That would be incorrect because we are allowing and testing both single documents and groups here.
I think also for single document it is good to have the test here instead of relying on content type tests like person.